resources/lib/utilities.py

```diff
@@ -412,6 +422,13 @@ def compareEpisodes(shows_col1, shows_col2, matchByTitleAndYear, watched=False,
                     if season in season_col2:
                         b = season_col2[season]
                         diff = list(set(a).difference(set(b)))
+                        for key in a:
+                            # update lastplayed in KODI if they don't match trakt
+                            if not key in b or a[key]['last_watched_at'] != b[key]['last_watched_at']:
```

Comment:
Not a 100% sure how to solve this. This will update any out of date timestamps not just ones to be removed.